types/node/process.d.ts

```diff
@@ -92,7 +92,7 @@ declare module 'process' {
             type DisconnectListener = () => void;
             type ExitListener = (code: number) => void;
             type RejectionHandledListener = (promise: Promise<unknown>) => void;
-            type UncaughtExceptionListener = (error: Error) => void;
+            type UncaughtExceptionListener = (error: Error, origin: string) => void;
```

Comment:
Please introduce a new type for this maybe `UncaughtExceptionOrigin = 'uncaughtException' | 'unhandledRejection'` and use it here.